independent-publisher-connectors/Google Photos/apiDefinition.swagger.json

```diff
@@ -0,0 +1,161 @@
+        "operationId": "CreateItems",
+        "parameters": [
+          {
+            "name": "mediaItemIds",
+            "default": "<string>",
+            "in": "query",
+            "type": "string",
+            "x-ms-summary": "Create Item",
+            "description": "Create Media Item - Photo, video etc",
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1/mediaItems:batchCreate": {},

```

Comment:
@juliamuiruri4 can you please explain why you have an empty paths? May I ask you please to remove all empty paths from your swagger file.